source/extensions/tracers/datadog/span.cc

```diff
@@ -137,11 +137,16 @@ void Span::setBaggage(absl::string_view, absl::string_view) {
 
 std::string Span::getTraceIdAsHex() const {
   if (!span_) {
-    return std::string{};
+    return EMPTY_STRING;
   }
   return absl::StrCat(absl::Hex(span_->id()));
 }
 
+std::string Span::getSpanIdAsHex() const {
+  // TODO not implemented
```

Comment:
Please add a name/PR# and consider adding some context on what needs to be added.

Reply:
I addressed this comment in [this commit](https://github.com/envoyproxy/envoy/pull/33909/commits/0639045524e6f1cc95a4526c50037a45989aa41d). To be completely truthful, I have no experience with Datadog, so I don't believe I have any more additional context to add. Please let me know if there's anything else I can add here - thanks!